packages/opencensus-exporter-stackdriver/package.json

```diff
@@ -49,12 +49,12 @@
     "nock": "^10.0.0",
     "nyc": "^13.0.0",
     "ts-node": "^7.0.1",
-    "typescript": "~3.2.0"
+    "typescript": "~3.1.0"
```

Comment:
This is known issue: https://github.com/googleapis/google-auth-library-nodejs/issues/519. Build is failing with 3.2.x.